plugins/leemons-plugin-academic-portfolio/frontend/src/components/ProgramSetup/SetupCourses/SetupCourses.js

```diff
@@ -230,7 +230,7 @@ const SetupCourses = ({
                         '{n}',
                         maxSubstageAbbreviation
                       )
-                    : 'Maximum ' + maxSubstageAbbreviation + ' characters';
+                    : `Maximum ${maxSubstageAbbreviation} characters`;
```

Comment:
Non translated label

Reply:
There are more in this file

Reply:
Clean most of the linter reported errors, included this one 😄